esmvalcore/preprocessor/_area.py

```diff
@@ -179,6 +180,18 @@ def meridional_statistics(cube, operator):
     raise ValueError(msg)
 
 
+def compute_area_weights(cube):
+    """Compute area weights."""
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            'ignore',
+            message="Using DEFAULT_SPHERICAL_EARTH_RADIUS.",
```

Comment:
@zklaus Is there any way to avoid assuming a spherical earth in this computation?

Reply:
According to [the iris docs](https://scitools-iris.readthedocs.io/en/latest/generated/api/iris/analysis/cartography.html#iris.analysis.cartography.area_weights) not at the moment.

Reply:
In that case, is this something we could check as part of the CMOR checks and issue the warning there?

Reply:
Yeah, I think that's a good idea. Should be a fairly low key warning probably, because it will be very common, but something that we can start nudging people towards.

Reply:
@zklaus Would you be willing to create an issue with some instructions, so we keep this on our to do list?

Reply:
For now, I turned the warning into a debug log message.

Reply:
I opened the issue at #1343.

Reply:
Thanks!